### PR TITLE
define parameters_get block and generator

### PR DESCRIPTION
**Pull request by mikeharv**
Core Blockly uses the same `variables_get` block for global variables as well as parameters. However, our levels expect a similar `parameters_get` block, which is found in the modal function editor.  These blocks generate the same code but have a different appearance. Compare the `direction` parameter with the `left` variable:
image

Due to concerns about stretching an already thin palette of accessible block colors, I'm hesitant to define a new block style for this block. Instead, for now, I've opted to re-use our "sprite_blocks" style, because a) sprites and parameters are so far never both exposed within the same lab, and b) the color seems similar enough. 

image

Note that this branch just defines the block appearance and code generation. The work to actually add them to the mini-toolbox (as shown above) is in another branch: https://github.com/code-dot-org/code-dot-org/pull/59180 

## Links

https://codedotorg.atlassian.net/browse/CT-625

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/blockly/addons/cdoVariables.ts [hunk 1]
```diff
@@ -6,6 +6,8 @@ import {BlocklyWrapperType} from '../types';
 export default function initializeVariables(
   blocklyWrapper: BlocklyWrapperType
 ) {
+  blocklyWrapper.JavaScript.forBlock.parameters_get =
```
Comment (molly-moen) [inline apps/src/blockly/addons/cdoVariables.ts:9]: what does this line do?

Comment (mikeharv) [reply apps/src/blockly/addons/cdoVariables.ts:9]: Tells Blockly to use the same generator function that it uses for standard variable getters.

Comment (molly-moen) [reply apps/src/blockly/addons/cdoVariables.ts:9]: Mind adding a comment explaining that? I could see it being confusing that we are doing things for parameters inside the variables function.

Comment (mikeharv) [reply apps/src/blockly/addons/cdoVariables.ts:9]: Yup, can do

**Review by molly-moen — APPROVED** [review]

**Pull request merged**